cobalt/browser/switches.cc

```diff
@@ -343,6 +343,10 @@ const char kQrCodeOverlayHelp[] =
     " and the number of locations can be overwritten by specifying it as the "
     " value of the command line parameter, like '--qr_code_overlay=6'.";
 
+const char kRasterizerType[] = "rasterizer_type";
+const char kRasterizerTypeHelp[] =
+    "Specifies the graphics rendering engine (direct-gles or skia) to use.";
```

Comment:
Tiny convenience nitpick: I don't want to type `direct-gles` every time, even though that's what it's called internally. I think `gles` or `gl` would be okay here, wdyt ?

Reply:
Right now we only have these two options direct-gles or skia. To even simplify this command line setting, I guess we can directly change this option to something like kEnableSkiaRasterizer, and make it a boolean? This will be sufficient for our use case.

Reply:
Take my words back. The change is smaller if we allow "skia" to set skia rasterizer and everything else as direct-gles rasterizer. This way, we still use direct-gles as the default renderer.

Reply:
Boolean may be a better setting here, yeah. Maybe it's still worth having the command-line setting be able to force it either way, regardless of what the default for given build is ? We might change the defaults sometime later

Reply:
Done.